app/src/main/java/com/jerboa/db/AppDB.kt

```diff
@@ -32,7 +32,7 @@ val APP_SETTINGS_DEFAULT = AppSettings(
     useCustomTabs = true,
     usePrivateTabs = false,
     secureWindow = false,
-    blurNSFW = true,
+    blurNSFW = 1,
```

Comment:
blur_nsfw is a `LocalUser` / back end setting, so it should probably not be here. Otherwise they're duped, and one is ignored.

Reply:
That setting existed before the backend option was added. I am not sure when it was added but we would need to implement a way to keep this backwards compatible with previous versions. So until then, it is better to just keep it local